Troubleshooting.md

```diff
@@ -0,0 +1,21 @@
+﻿# Build Troubleshooting
+Below are some tips for troubleshooting common issues.
+
+### Docker-Related Errors
+If you receive an error similar to:
```

Comment:
Our tests are much more resilient to docker failures now, I think we can remove this section.